src/App.js

```diff
@@ -1,13 +1,28 @@
 import "./App.css";
+import "./Mode.css";
 import Game from "./Game/Game";
+import { useState } from "react";
 
 function App() {
   let wel = "Welcome to TicTacToe!";
+  const [darkMode, setDarkMode] = useState(false);
   return (
-    <>
-    <div className="wel">{wel}</div>
-    <Game />
-    </>
+    <div className={darkMode ? "dark-mode" : "light-mode"}>
+      <div className="container">
+        <span style={{ color: darkMode ? "grey" : "yellow" }}>☀︎</span>
```

Comment:
It is better to remove the inline style and create a separate object for style for better code understanding